src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/fetchMarkup/index.test.js

```diff
@@ -0,0 +1,5 @@
+// import fetchMarkup from '.';
+
+describe('fetchMarkup', () => {
+  it('should work ?', () => {});
```

Comment:
Obviously aware this isn't finished but where possible it would be good to pull tests down from `index.test.js` where they relate to each of these new components you've extracted

Reply:
Since all modules will now have their tests I was actually going to mock every function call(external dependency) in the tests for `index.test.js` so changing logic in any of those don't actually break this as I was getting that a lot. Having this still the way it is was just for a sanity check that everything still works as expected

Reply:
> I was actually going to mock every function call(external dependency) in the tests for index.js 

That's an approach I used to always take when testing components, it's great for focusing the testing on the component itself rather than its dependencies. An approach I've started to go for is having a smaller set of tests that don't mock any dependencies for the top level 'orchestration' component (`index.js`), but make sure everything works together; these are integration tests but at a component level rather than a page level like when have elsewhere. Then all the dependent components have proper 'unit tests' with most dependencies mocked to keep the tests focused only on the components itself.

I think your approach sounds good, just thought I'd share what I've been thinking lately.